lib/jxl/render_pipeline/stage_gaborish.cc

```diff
@@ -53,25 +53,25 @@ class GaborishStage : public RenderPipelineStage {
       const auto w0 = Set(d, weights_[3 * c + 0]);
       const auto w1 = Set(d, weights_[3 * c + 1]);
       const auto w2 = Set(d, weights_[3 * c + 2]);
-      // Since GetInputRow(input_rows, c, {-1, 0, 1}) + kRenderPipelineXOffset
+      // Since GetInputRow(input_rows, c, {-1, 0, 1})
```

Comment:
I think this comment can be reflowed now, right?

Reply:
Ahhh autoformat failing me :D